### PR TITLE
Addition of a new fan for parking.

### PR DESCRIPTION
**Pull request by ghairapetian**

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 1]
```diff
@@ -7171,3 +7171,25 @@ FPHC:
   uses:
   - heating_water_valve_percentage_command
   - chilled_water_valve_percentage_command
+
+COC2DSP:
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:7175]: Please revise name to `CO2C2XDSP`

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 2]
```diff
@@ -7171,3 +7171,25 @@ FPHC:
...
+  is_abstract: true
+  uses:
+  - high_limit_zone_air_co_concentration_setpoint
+  - low_limit_zone_air_co_concentration_setpoint
+  - zone_air_co_concentration_sensor_1
+  - zone_air_co_concentration_sensor_2
+  implements:
+  - CONTROL
+
+NO2C2DSP:
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:7186]: Please revise name to `NO2C2XDSP`

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml (hunk 1 above)
Comment (ghairapetian) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml]: Done.

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml (hunk 2 above)
Comment (ghairapetian) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml]: Done.

**Pull request merged**